templates/default/sentinel.conf.erb

```diff
@@ -15,6 +15,8 @@ syslog-facility <%= @syslogfacility %>
 # The port that this sentinel instance will run on
 port <%=@sentinel_port%>
 
+<% @masters.each do |current| %>
```

Comment:
I think you're probably right. It does end up being a huge number of lines of just comments. I'll trim it down and push one more commit.